<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormUploader</class>
 <widget class="QWidget" name="FormUploader">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Jura</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>GüntherUploader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ui.qrc">
    <normaloff>:/general/gunther icono.png</normaloff>:/general/gunther icono.png</iconset>
  </property>
  <widget class="QLabel" name="LogoMixcloud">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>290</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>290</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>290</width>
     <height>48</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>290</width>
     <height>48</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/general/mixcloud-logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jura</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GüntherUploader</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>96</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del podcast:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textPodcastName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textDescripcion">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Descripción:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Subir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCerrar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textStatus">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>200</y>
     <width>448</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="plainText">
    <string>Presione &quot;Subir&quot; para iniciar el upload, o &quot;Cerrar&quot; para cancelarlo...
</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>177</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections/>
</ui>
